packages/sanity/src/form/patch/types.ts

```diff
@@ -1,6 +1,11 @@
-/* eslint-disable camelcase */
+import type {Path} from '@sanity/types'
 
-import {Path} from '@sanity/types'
+/**
+ * @alpha
+ */
+export interface FormPatchBase {
+  patchType: symbol
+}
```

Comment:
Should we instead mark the property as `@internal`?

React calls these `$$typeof` – maybe a convention we can use too?

```suggestion
/**
 * @public
 */
export interface FormPatchBase {
  /** @internal */
  $$typeof: symbol
}
```

Reply:
I'm alright with both of those changes - lets hear if @bjoerge agrees

Reply:
+1 to making it internal. Not sure about adopting `$$typeof` - feels too much like relying on an internal React convention, and worried it might add confusion. I.e. during debugging, `$$typeof` provides a quick way to eyeball the react boundary (e.g. is this thing coming from React or our code?). I'd say keep it as `patchType`, but yeah, keep it `@internal`